.github/workflows/ci.yml

```diff
@@ -487,6 +487,49 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+          cache: 'maven'
+      - name: Maven Install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          # GIB needs to be explicitly disabled, because the gib profile enables it, but the trino-server module requires all of its dependencies to be built
+          $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -Dgib.disable -pl '!:trino-docs,!:trino-server-rpm'
+      - name: Server artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: server tarball
+          path: |
+            core/trino-server/target/*.tar.gz
+          retention-days: 1
+      - name: Product tests launcher artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: product tests launcher
+          path: |
+            testing/trino-product-tests-launcher/target/*.jar
+          retention-days: 1
+      - name: Product tests artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: product tests
+          path: |
+            testing/trino-product-tests/target/*.jar
```

Comment:
we need only the final jar, what other things is this uploading?

Reply:
Right, I wasn't sure. I checked that the launcher expects the executable jar.